src/types/CaseListQueryParams.ts

```diff
@@ -2,13 +2,18 @@ export type QueryOrder = "asc" | "desc" | undefined
 
 export type Reason = "Triggers" | "Exceptions" | "Bails"
 
-export type NamedCourtDateRanges = "Today" | undefined
+export type CaseAge = "Today" | "Yesterday" | "Day 2" | "Day 3"
```

Comment:
I think we need more case ages than these, the design has a scroll bar which would show more case ages. You can check with the designers for what case ages we want as options but I think we'll want at least 14 days